src/transformStateWithClones.js

```diff
@@ -7,7 +7,29 @@
  * @return {Object[]}
  */
 function transformStateWithClones(state, actions) {
-  // write code here
+  const newState = { ...state };
+  const result = [];
```

Comment:
result/res/arr/elem/obj/etc are ban names, the name should describe the variable